beta-src/src/state/game/game-api-slice.ts

```diff
@@ -212,6 +212,10 @@ const gameApiSlice = createSlice({
       .addCase(fetchGameData.pending, (state) => {
         state.apiStatus = "loading";
         state.transition = true;
+        state.ordersMeta = {};
+        if (state.overview.phase !== "Builds") {
+          state.mustDestroyUnitsBuildPhase = false;
+        }
```

Comment:
This makes more sense to move it to  `fetchGameOverview.pending` since this is toggled in `fetchGameOverview.fulfilled, fetchGameOverviewFulfilled)`

Reply:
I believe it is in pending